Whiskey/Tasks/PublishNodeModule.ps1

```diff
@@ -89,19 +89,19 @@ function Publish-WhiskeyNodeModule
 
         Copy-Item -Path $originalPackageJsonPath -Destination $backupPackageJsonPath
         Invoke-WhiskeyNpmCommand -Name 'version' `
-                                 -ArgumentList $TaskContext.Version.SemVer2NoBuildMetadata, '--no-git-tag-version', '--allow-same-version' `
+                                 -ArgumentList $TaskContext.Version.SemVer2NoBuildMetadata, '--no-git-tag-version' `

```

Comment:
I don't understand how this fixes the bug. Isn't the issue with the `npm publish` command?

Reply:
No the issue is with the `npm version` command which is responsible for bumping a package version: https://docs.npmjs.com/cli/v10/commands/npm-version#allow-same-version

Reply:
The `npm version` command just updates the version number in the package.json file on-disk. I don't think it knows anything about what versions have already been published to the registry.

Reply:
Ok I understand now. I'll use the `npm view` command to get latest published version of a package and compare that to the version in the Task Context. That should do the job.

Reply:
Turns out this was an issue with the setting in our private node module registry that allowed over writing of packages. Closing out PR.